WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Split Shipments/WooShippingSplitShipmentsDetailView.swift

```diff
@@ -0,0 +1,76 @@
+import SwiftUI
+
+struct WooShippingSplitShipmentsDetailView: View {
+    @Environment(\.dismiss) private var dismiss
+
+    let viewModel: WooShippingSplitShipmentsViewModel
+
+    var body: some View {
+        NavigationView {
+            ScrollView {
+                VStack(alignment: .leading, spacing: Layout.contentPadding) {
+                    AdaptiveStack {
```

Comment:
Nit: we should set the horizontal alignment to `.leading` to make the view look neater when big font sizes are set:

<img src="https://github.com/user-attachments/assets/2e898510-3176-4d21-bc62-a3f0e654ffec" width=320 />

Reply:
Thank you! Done in https://github.com/woocommerce/woocommerce-ios/pull/15369/commits/304725ac06628ce22f752142c14502efd1d0f34d